api/src/tests/invalid_post_request_test.rs

```diff
@@ -44,22 +44,6 @@ async fn test_invalid_script_function_argument_address_string() {
     response_error_msg(req, current_function_name!()).await;
 }
 
-#[tokio::test]
-async fn test_invalid_script_function_argument_hex_encoded_bytes_type() {
-    let mut req = signing_message_request();
-    req["payload"]["arguments"][1] = json!({});
```

Comment:
these tests are deleted since arg 1 for auth_key_pre_image doesn't exist after the change